peekingduck/configs/heuristic/fps.yml

```
@@ -0,0 +1,6 @@
+input: ["pipeline_end"]
+output: ["fps"]
+
+fps_log_display: True   # Boolean to display logging of fps moving average
+fps_log_freq: 30   # Logging frequency of FPS every n frames in cli
+dampen_fps: True   # Returns average FPS if True, instantaneous if False
```

If this is too long and fails pylint then don't worry about it
```suggestion
dampen_fps: True   # Returns FPS averaged over last 10 frames if True, instantaneous if False
```

---

Updated